spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -86,8 +86,8 @@ def compute_firing_rates(waveform_extractor):
     return firing_rates
 
 
-def compute_presence_ratios(waveform_extractor, bin_duration_s=60):
-    """Calculate the presence ratio, representing the fraction of time the unit is firing.
+def compute_presence_ratios(waveform_extractor, bin_duration_s=60.0, mean_fr_ratio_thres=0.0):
```

Comment:
```suggestion
def compute_presence_ratios(waveform_extractor, bin_duration_s=60.0, mean_fr_ratio_thresh=0.0):
```

@TomBugnon I think that in other places in the API we use `thresh` instead of `thres` for abbreviating `threshold`. Could you rename the parameter?